### PR TITLE
Longest Common Subsequence in C++

### PR DESCRIPTION
**Pull request by SrijitaSarkar99**
Added markdown file containing code snippet to find the longest common subsequence in c++

### CODE REVIEW

File: snippets/c++/Longest_common_subsequence/lcs.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+	lcs(str1, str2, m, n); 
+	return 0; 
+} 
+
+```
+
+## Output
+ 
+![LCS image](LCS_example.png?raw=true "LCS_example")   
+
```
Comment (kaustubhgupta) [inline snippets/c++/Longest_common_subsequence/lcs.md]: This program is fairly simple so no need to put screenshot here. Follow our I/O block implementation

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

File: snippets/c++/Longest_common_subsequence/lcs.md (hunk 1 above)
Comment (SrijitaSarkar99) [reply snippets/c++/Longest_common_subsequence/lcs.md]: @kaustubhgupta, I updated the markdown file. Please review.

**Comment by sakshamarora1** [comment]
Move `lcs.md` outside of the folder as there is no need for an extra folder level. Also rename the the file to what the folder name was. Apart from that, everything looks good!

File: snippets/c++/Longest_common_subsequence/lcs.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
+# Longest Common Subsequence
```
Comment (kaustubhgupta) [inline snippets/c++/Longest_common_subsequence/lcs.md]: Now make the changes suggested by Saksham and your PR will be merged :-)

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

File: snippets/c++/Longest_common_subsequence/lcs.md (hunk 2 above)
Comment (SrijitaSarkar99) [reply snippets/c++/Longest_common_subsequence/lcs.md]: @kaustubhgupta @sakshamarora1 
I made the suggested changes. Please review :)

**Review by kaustubhgupta — APPROVED** [review]

**Pull request merged**